src/NSubstitute/NSubstitute.csproj

```diff
@@ -49,10 +48,10 @@
   </ItemGroup>
   
   <ItemGroup>
-    <PackageReference Include="Castle.Core" Version="5.1.1-*" />
+    <PackageReference Include="Castle.Core" Version="5.1.1" />
```

Comment:
**question:** can locking in to a specific `Castle.Core` version cause issues for projects that also use `Castle.Core` internally (i.e. not just via nsub?)

Reply:
It the same:
```<PackageReference Include="Castle.Core" Version="5.1.1-*" />```
and
```<PackageReference Include="Castle.Core" Version="5.1.1" />```
have the same beaviour

if you need specify exact version you need to use square brakets like this
```<PackageReference Include="Castle.Core" Version="[5.1.1]" />```

also is possible set specific version range
```<PackageReference Include="Hangfire.Core" Version="[1.7.0,1.9.0)" />```

more info: https://learn.microsoft.com/en-us/nuget/concepts/package-versioning?tabs=semver20sort#version-ranges

this change it just remove redunant ```-*``` syntax, behavior it the same

Reply:
Thanks for this information. 🙇 
I've been working on other platforms for a while so my knowledge here is very patchy 🤦 . I appreciate your patience explaining this. :)